### PR TITLE
Bumps core for production geocoder caching

### PR DESCRIPTION
**Pull request by benlovell**

### CODE REVIEW

**Comment by lshepstone** [comment]
:+1: Sure.

File: Gemfile.lock [hunk 1]
```diff
@@ -152,12 +152,13 @@ GEM
...
     mail (2.6.3)
       mime-types (>= 1.16, < 3)
-    mas-rad_core (0.0.17)
+    mas-rad_core (0.0.19)
       active_model_serializers
       geocoder
       http
       pg
       rails (~> 4.2.0)
+      redis
```
Comment (esbel) [inline Gemfile.lock]: redis is locked at 3.2.0, while on consumer it's 3.2.1, is it worth bumping as well, just for the sake of consistency?

**Comment by esbel** [comment]
:+1:, regardless of Redis version

File: Gemfile.lock (hunk 1 above)
Comment (benlovell) [inline Gemfile.lock]: Seems as though that's down to `sidekiq`, since it's just one patch level I think I'll leave it for now.

Comment (esbel) [inline Gemfile.lock]: ok, no worries

**Pull request merged**